biz.aQute.bndlib/src/aQute/bnd/osgi/Constants.java

```diff
@@ -544,6 +544,7 @@ public interface Constants {
 	String		STATIC_ATTRIBUTE							= "static";
 	String		SUBSTITUTE_ATTRIBUTE						= "substitute";
 	String		MODULE_INFO_CLASS							= "module-info.class";
+	String		JPMS_MULTI_RELEASE							= "-jpms-multi-release";
```

Comment:
There is really no such thing as "jpms-multi-release"  see:

https://docs.oracle.com/en/java/javase/16/docs/specs/jar/jar.html#modular-multi-release-jar-files

> A modular multi-release JAR file is a multi-release JAR file that has a module descriptor, module-info.class, in the top-level directory (as for a [modular](https://docs.oracle.com/en/java/javase/16/docs/specs/jar/jar.html#modular-jar-files) JAR file), or directly in a versioned directory.
> 
> ...
> 
> **A module descriptor is generally treated no differently to any other class or resource file**. A module descriptor may be present under a versioned area but not present under the top-level directory.

Reply:
this is an option for bnd to activate the plugin